schema/relationship.go

```diff
@@ -32,6 +33,8 @@ type Relationships struct {
 	Relations map[string]*Relationship
 
 	EmbeddedRelations map[string]*Relationships
+
+	Mux *sync.RWMutex
```

Comment:
thank you for your PR, let’s use a non-pointer value to avoid potential nil panics.

Reply:
Done!